src/ucp/core/ucp_worker.c

```diff
@@ -328,11 +328,19 @@ static void ucp_worker_wakeup_cleanup(ucp_worker_h worker)
     }
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_worker_iface_test_event_fd_support(const ucp_worker_iface_t *wiface)
+{
+    return (wiface->attr.cap.event_flags & UCT_IFACE_FLAG_EVENT_FD) &&
+           (wiface->attr.cap.event_flags & UCP_WORKER_UCT_ALL_EVENT_CAP_FLAGS);
```

Comment:
ucs_test_all_flags(wiface->attr.cap.event_flags, UCT_IFACE_FLAG_EVENT_FD || UCP_WORKER_UCT_ALL_EVENT_CAP_FLAGS)

Reply:
👍  but single `|`

Reply:
we don't want to support all flags set in `UCP_WORKER_UCT_ALL_EVENT_CAP_FLAGS`, at least one should be supported

Reply:
I see

Reply:
> we don't want to support all flags set in `UCP_WORKER_UCT_ALL_EVENT_CAP_FLAGS`, at least one should be supported

pls add a comment it's not clear from code